==194894== Memcheck, a memory error detector
==194894== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==194894== Using Valgrind-3.21.0 and LibVEX; rerun with -h for copyright info
==194894== Command: bin/mandatory/fdf data/mars.fdf
==194894== 
==194894== 
==194894== HEAP SUMMARY:
==194894==     in use at exit: 310,971 bytes in 3,042 blocks
==194894==   total heap usage: 2,924,265 allocs, 2,921,223 frees, 1,698,477,990 bytes allocated
==194894== 
==194894== 6 bytes in 1 blocks are still reachable in loss record 1 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4A6ABED: strdup (strdup.c:42)
==194894==    by 0x4C248CC: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 8 bytes in 1 blocks are still reachable in loss record 2 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C30CA9: _XimOpenIM (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x48807BD: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 9 bytes in 1 blocks are still reachable in loss record 3 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1FF2B: _XlcDefaultMapModifiers (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24379: XSetLocaleModifiers (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x48807AB: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 12 bytes in 1 blocks are still reachable in loss record 4 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4A6ABED: strdup (strdup.c:42)
==194894==    by 0x4C1FBF7: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 16 bytes in 1 blocks are still reachable in loss record 5 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x1122E6: ft_gnl_init_struct (ft_gnl_utils.c:44)
==194894==    by 0x112072: get_arch_data (ft_gnl.c:105)
==194894==    by 0x1120C9: ft_gnl (ft_gnl.c:117)
==194894==    by 0x10AFB0: ft_mx_load_file (ft_mx_load_file.c:120)
==194894==    by 0x10A816: main (fdf_main.c:29)
==194894== 
==194894== 16 bytes in 1 blocks are still reachable in loss record 6 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C23CA8: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 17 bytes in 1 blocks are still reachable in loss record 7 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F534: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 18 bytes in 1 blocks are still reachable in loss record 8 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F628: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930EDE9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 20 bytes in 1 blocks are still reachable in loss record 9 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F0C5: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 23 bytes in 1 blocks are still reachable in loss record 10 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x9429112: void std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >::_M_realloc_insert<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>(__gnu_cxx::__normal_iterator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F659: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9306FFA: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 24 bytes in 1 blocks are still reachable in loss record 11 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1DFE9: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FB89: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 24 bytes in 1 blocks are still reachable in loss record 12 of 97
==194894==    at 0x484ABC0: realloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1EFE0: _XlcResolveLocaleName (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24A07: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 24 bytes in 1 blocks are still reachable in loss record 13 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930FC24: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 27 bytes in 1 blocks are still reachable in loss record 14 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD49: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 27 bytes in 1 blocks are still reachable in loss record 15 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F628: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD49: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 28 bytes in 1 blocks are still reachable in loss record 16 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F628: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD03: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 29 bytes in 1 blocks are still reachable in loss record 17 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F628: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD26: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 32 bytes in 1 blocks are still reachable in loss record 18 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x40137DD: malloc (rtld-malloc.h:56)
==194894==    by 0x40137DD: allocate_dtv_entry (dl-tls.c:678)
==194894==    by 0x40137DD: allocate_and_init (dl-tls.c:703)
==194894==    by 0x40137DD: tls_get_addr_tail (dl-tls.c:901)
==194894==    by 0x401758B: __tls_get_addr (tls_get_addr.S:55)
==194894==    by 0xC42141E: llvm::initializeCodeGenerationPass(llvm::PassRegistry&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0xC36391C: polly::initializePollyPasses(llvm::PassRegistry&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x933C7F7: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 40 bytes in 1 blocks are still reachable in loss record 19 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C20453: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 40 bytes in 1 blocks are still reachable in loss record 20 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930EDE9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 44 bytes in 1 blocks are still reachable in loss record 21 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4027D7E: malloc (rtld-malloc.h:56)
==194894==    by 0x4027D7E: strdup (strdup.c:42)
==194894==    by 0x4008EEB: _dl_map_object (dl-load.c:2198)
==194894==    by 0x400D058: dl_open_worker_begin (dl-open.c:534)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894==    by 0x4A4F066: _dlerror_run (dlerror.c:138)
==194894== 
==194894== 44 bytes in 1 blocks are still reachable in loss record 22 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92FCE73: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 47 bytes in 1 blocks are still reachable in loss record 23 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930FC24: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 47 bytes in 5 blocks are still reachable in loss record 24 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18C02: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894== 
==194894== 48 bytes in 1 blocks are still reachable in loss record 25 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C141B9: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0E658: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F08E: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 48 bytes in 3 blocks are still reachable in loss record 26 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C2002D: _XlcAddLoader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C200A5: _XlcInitLoader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C70: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 49 bytes in 1 blocks are still reachable in loss record 27 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9324DB9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 51 bytes in 1 blocks are still reachable in loss record 28 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x931417B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 53 bytes in 2 blocks are still reachable in loss record 29 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x931ACF4: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 54 bytes in 1 blocks are still reachable in loss record 30 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9313144: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 56 bytes in 2 blocks are still reachable in loss record 31 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9313144: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 57 bytes in 2 blocks are still reachable in loss record 32 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD03: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 62 bytes in 3 blocks are still reachable in loss record 33 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F89A: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9306FFA: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 62 bytes in 4 blocks are still reachable in loss record 34 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18A62: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2163E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 64 bytes in 1 blocks are still reachable in loss record 35 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4BFC235: _XrmInternalStringToQuark (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0DE53: XrmInitialize (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F268: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 64 bytes in 4 blocks are still reachable in loss record 36 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18929: _XlcAddCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C21662: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 72 bytes in 1 blocks are still reachable in loss record 37 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F863: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F604: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 72 bytes in 1 blocks are still reachable in loss record 38 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F863: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F64B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 72 bytes in 1 blocks are still reachable in loss record 39 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F863: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930FC24: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 78 bytes in 2 blocks are still reachable in loss record 40 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x931ACF4: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 78 bytes in 4 blocks are still reachable in loss record 41 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C189F6: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2163E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 80 bytes in 3 blocks are still reachable in loss record 42 of 97
==194894==    at 0x484ABC0: realloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C21598: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894== 
==194894== 85 bytes in 2 blocks are still reachable in loss record 43 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F604: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 92 bytes in 2 blocks are still reachable in loss record 44 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9306FFA: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 94 bytes in 2 blocks are still reachable in loss record 45 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F0C5: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 95 bytes in 4 blocks are still reachable in loss record 46 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x9429112: void std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >::_M_realloc_insert<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>(__gnu_cxx::__normal_iterator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F659: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9308B0A: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 96 bytes in 4 blocks are still reachable in loss record 47 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F628: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F534: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 104 bytes in 1 blocks are still reachable in loss record 48 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93DFC91: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9422365: llvm::ManagedStaticBase::RegisterManagedStatic(void* (*)(), void (*)(void*)) const (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93DDE76: llvm::DebugCounter::instance() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EE1F: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92FCE73: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 104 bytes in 3 blocks are still reachable in loss record 49 of 97
==194894==    at 0x484ABC0: realloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C11B75: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C13524: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 104 bytes in 13 blocks are still reachable in loss record 50 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C21679: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894== 
==194894== 114 bytes in 3 blocks are still reachable in loss record 51 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932AD03: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 128 bytes in 1 blocks are still reachable in loss record 52 of 97
==194894==    at 0x484ABC0: realloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1E2B4: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C21137: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 128 bytes in 4 blocks are still reachable in loss record 53 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4002055: malloc (rtld-malloc.h:56)
==194894==    by 0x4002055: _dl_close_worker.part.0.isra.0 (dl-close.c:371)
==194894==    by 0x4002631: _dl_close_worker (dl-close.c:120)
==194894==    by 0x4002631: _dl_close (dl-close.c:809)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894==    by 0x4A4F066: _dlerror_run (dlerror.c:138)
==194894==    by 0x4A4ED95: dlclose@@GLIBC_2.34 (dlclose.c:31)
==194894==    by 0x48792B2: ??? (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x114796: mlx_terminate (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FEAA: fdf_start_gui (fdf_start_gui.c:41)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 140 bytes in 3 blocks are still reachable in loss record 54 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF8E4E4E: std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_assign(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x981F574: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930FC67: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 168 bytes in 1 blocks are still reachable in loss record 55 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1E01E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FB89: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 176 bytes in 1 blocks are still reachable in loss record 56 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1E004: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FB89: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 182 bytes in 3 blocks are still reachable in loss record 57 of 97
==194894==    at 0x484ABC0: realloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C11BA8: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C13524: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 288 bytes in 4 blocks are still reachable in loss record 58 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C189CA: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2163E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 320 bytes in 2 blocks are still reachable in loss record 59 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0E31: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9422365: llvm::ManagedStaticBase::RegisterManagedStatic(void* (*)(), void (*)(void*)) const (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D4B98: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9422365: llvm::ManagedStaticBase::RegisterManagedStatic(void* (*)(), void (*)(void*)) const (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93CDCF6: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C489D: llvm::cl::getGeneralCategory() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x954D316: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92F8357: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894== 
==194894== 352 bytes in 1 blocks are still reachable in loss record 60 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D4AD4: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9422365: llvm::ManagedStaticBase::RegisterManagedStatic(void* (*)(), void (*)(void*)) const (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93CDCF6: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C489D: llvm::cl::getGeneralCategory() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x954D316: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92F8357: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 400 bytes in 1 blocks are still reachable in loss record 61 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xA65F7D1: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9422365: llvm::ManagedStaticBase::RegisterManagedStatic(void* (*)(), void (*)(void*)) const (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0xA65D2BA: llvm::PassManagerBuilder::addGlobalExtension(llvm::PassManagerBuilder::ExtensionPointTy, std::function<void (llvm::PassManagerBuilder const&, llvm::legacy::PassManagerBase&)>) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x933C833: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 432 bytes in 6 blocks are still reachable in loss record 62 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F863: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92FCE73: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 633 bytes in 49 blocks are still reachable in loss record 63 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18A62: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18DDA: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 761 bytes in 17 blocks are still reachable in loss record 64 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930EE30: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 787 bytes in 85 blocks are still reachable in loss record 65 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C11C7E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C13524: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 792 bytes in 11 blocks are still reachable in loss record 66 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x981F863: std::_Rb_tree_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int>, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> > >::_M_emplace_hint_unique<std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>, std::tuple<> >(std::_Rb_tree_const_iterator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, unsigned int> >, std::piecewise_construct_t const&, std::tuple<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>&&, std::tuple<>&&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F812: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F5D9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930FC67: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 800 bytes in 1 blocks are still reachable in loss record 67 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x400C6FC: malloc (rtld-malloc.h:56)
==194894==    by 0x400C6FC: add_to_global_resize (dl-open.c:152)
==194894==    by 0x400D6BF: dl_open_worker_begin (dl-open.c:718)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894==    by 0x4A4F066: _dlerror_run (dlerror.c:138)
==194894==    by 0x4A4F620: dlopen_implementation (dlopen.c:71)
==194894==    by 0x4A4F620: dlopen@@GLIBC_2.34 (dlopen.c:81)
==194894== 
==194894== 800 bytes in 50 blocks are still reachable in loss record 68 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18929: _XlcAddCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18DEA: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 850 bytes in 23 blocks are still reachable in loss record 69 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4027D7E: malloc (rtld-malloc.h:56)
==194894==    by 0x4027D7E: strdup (strdup.c:42)
==194894==    by 0x4016805: _dl_load_cache_lookup (dl-cache.c:515)
==194894==    by 0x40091DB: _dl_map_object (dl-load.c:2132)
==194894==    by 0x4002894: openaux (dl-deps.c:64)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4002CC6: _dl_map_object_deps (dl-deps.c:232)
==194894==    by 0x400D0C0: dl_open_worker_begin (dl-open.c:594)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 894 bytes in 24 blocks are still reachable in loss record 70 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x400C2C8: malloc (rtld-malloc.h:56)
==194894==    by 0x400C2C8: _dl_new_object (dl-object.c:199)
==194894==    by 0x400752E: _dl_map_object_from_fd (dl-load.c:1053)
==194894==    by 0x4008F70: _dl_map_object (dl-load.c:2265)
==194894==    by 0x4002894: openaux (dl-deps.c:64)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4002CC6: _dl_map_object_deps (dl-deps.c:232)
==194894==    by 0x400D0C0: dl_open_worker_begin (dl-open.c:594)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 1,024 bytes in 1 blocks are still reachable in loss record 71 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x94290CA: void std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >::_M_realloc_insert<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&>(__gnu_cxx::__normal_iterator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F659: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F36B: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9324DB9: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 1,028 bytes in 50 blocks are still reachable in loss record 72 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C189F6: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18DDA: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 1,280 bytes in 16 blocks are still reachable in loss record 73 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C1E285: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C21137: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 1,360 bytes in 85 blocks are still reachable in loss record 74 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C11C5D: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C13524: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 1,792 bytes in 28 blocks are still reachable in loss record 75 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18EA6: _XlcSetConverter (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18FC0: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894== 
==194894== 1,799 bytes in 42 blocks are still reachable in loss record 76 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930709A: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 2,048 bytes in 1 blocks are still reachable in loss record 77 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C2053A: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 2,136 bytes in 1 blocks are still reachable in loss record 78 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C203E5: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894== 
==194894== 2,358 bytes in 176 blocks are still reachable in loss record 79 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4A6ABED: strdup (strdup.c:42)
==194894==    by 0x4C13338: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 2,631 bytes in 50 blocks are still reachable in loss record 80 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C18B5F: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894== 
==194894== 3,520 bytes in 88 blocks are still reachable in loss record 81 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C13320: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1357E: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20347: _XlcCreateLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C24824: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 3,600 bytes in 50 blocks are still reachable in loss record 82 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4C189CA: _XlcCreateDefaultCharSet (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18DDA: _XlcAddCT (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C18F77: _XlcInitCTInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C246F6: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C16: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894== 
==194894== 4,064 bytes in 1 blocks are still reachable in loss record 83 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x400A6FA: calloc (rtld-malloc.h:44)
==194894==    by 0x400A6FA: do_lookup_unique (dl-lookup.c:272)
==194894==    by 0x400A6FA: do_lookup_x (dl-lookup.c:505)
==194894==    by 0x400AC47: _dl_lookup_symbol_x (dl-lookup.c:793)
==194894==    by 0x400FD11: resolve_map (dl-reloc.c:190)
==194894==    by 0x400FD11: elf_machine_rela (dl-machine.h:271)
==194894==    by 0x400FD11: elf_dynamic_do_Rela (do-rel.h:147)
==194894==    by 0x400FD11: _dl_relocate_object (dl-reloc.c:301)
==194894==    by 0x400D246: dl_open_worker_begin (dl-open.c:704)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894== 
==194894== 4,096 bytes in 1 blocks are still reachable in loss record 84 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4BFC087: _XrmInternalStringToQuark (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0DE53: XrmInitialize (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F268: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 4,608 bytes in 1 blocks are still reachable in loss record 85 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93DEBEB: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981F40E: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x981EF3C: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92FCE73: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 5,294 bytes in 133 blocks are still reachable in loss record 86 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92F8357: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 5,394 bytes in 128 blocks are still reachable in loss record 87 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x930F6A2: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 8,176 bytes in 1 blocks are still reachable in loss record 88 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4BFAE85: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4BFC20A: _XrmInternalStringToQuark (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0DE53: XrmInitialize (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F268: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894==    by 0x115851: mlx_init (in /home/lluque/011_fdf/repo_github/bin/mandatory/fdf)
==194894==    by 0x10FD02: fdf_start_gui (fdf_start_gui.c:21)
==194894==    by 0x10A8B5: main (fdf_main.c:36)
==194894== 
==194894== 8,176 bytes in 1 blocks are still reachable in loss record 89 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4BFC3BC: _XrmInternalStringToQuark (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C20131: _XlcGetLocaleDataBase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C21474: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C22C22: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C1FBAB: _XlcCreateLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C402A8: _XlcUtf8Loader (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C23C96: _XOpenLC (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C2469E: _XrmInitParseInfo (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0ECF7: ??? (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x4C0F061: XrmGetStringDatabase (in /usr/lib/x86_64-linux-gnu/libX11.so.6.4.0)
==194894==    by 0x487F2CA: glfwInit (in /usr/lib/x86_64-linux-gnu/libglfw.so.3.3)
==194894== 
==194894== 10,107 bytes in 246 blocks are still reachable in loss record 90 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x9308B97: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 10,786 bytes in 277 blocks are still reachable in loss record 91 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x92FCEB0: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 11,304 bytes in 23 blocks are still reachable in loss record 92 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x4015F8B: calloc (rtld-malloc.h:44)
==194894==    by 0x4015F8B: _dl_check_map_versions (dl-version.c:280)
==194894==    by 0x400D2BC: dl_open_worker_begin (dl-open.c:602)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894==    by 0x4A4F066: _dlerror_run (dlerror.c:138)
==194894==    by 0x4A4F620: dlopen_implementation (dlopen.c:71)
==194894==    by 0x4A4F620: dlopen@@GLIBC_2.34 (dlopen.c:81)
==194894== 
==194894== 24,083 bytes in 615 blocks are still reachable in loss record 93 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932ADA0: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 26,182 bytes in 615 blocks are still reachable in loss record 94 of 97
==194894==    at 0x4845FA3: operator new(unsigned long) (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x93D0D9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93131A3: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894== 
==194894== 29,572 bytes in 24 blocks are still reachable in loss record 95 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x400BF9C: calloc (rtld-malloc.h:44)
==194894==    by 0x400BF9C: _dl_new_object (dl-object.c:92)
==194894==    by 0x400752E: _dl_map_object_from_fd (dl-load.c:1053)
==194894==    by 0x4008F70: _dl_map_object (dl-load.c:2265)
==194894==    by 0x4002894: openaux (dl-deps.c:64)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4002CC6: _dl_map_object_deps (dl-deps.c:232)
==194894==    by 0x400D0C0: dl_open_worker_begin (dl-open.c:594)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C569: dl_open_worker (dl-open.c:784)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894== 
==194894== 49,164 bytes in 1 blocks are still reachable in loss record 96 of 97
==194894==    at 0x484A993: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0x944B1CA: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x944B907: llvm::StringMapImpl::RehashTable(unsigned int) (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0DDC: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93D0F9D: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x93C290E: llvm::cl::Option::addArgument() (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x932DF8E: ??? (in /usr/lib/x86_64-linux-gnu/libLLVM-15.so.1)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894== 
==194894== 73,728 bytes in 1 blocks are still reachable in loss record 97 of 97
==194894==    at 0x4845828: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==194894==    by 0xF83F491: ??? (in /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32)
==194894==    by 0x400536D: call_init.part.0 (dl-init.c:90)
==194894==    by 0x4005472: call_init (dl-init.c:136)
==194894==    by 0x4005472: _dl_init (dl-init.c:137)
==194894==    by 0x4001561: _dl_catch_exception (dl-catch.c:211)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:810)
==194894==    by 0x400C5C5: dl_open_worker (dl-open.c:773)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x400C91B: _dl_open (dl-open.c:886)
==194894==    by 0x4A4F56B: dlopen_doit (dlopen.c:56)
==194894==    by 0x40014DC: _dl_catch_exception (dl-catch.c:237)
==194894==    by 0x4001602: _dl_catch_error (dl-catch.c:256)
==194894==    by 0x4A4F066: _dlerror_run (dlerror.c:138)
==194894== 
==194894== LEAK SUMMARY:
==194894==    definitely lost: 0 bytes in 0 blocks
==194894==    indirectly lost: 0 bytes in 0 blocks
==194894==      possibly lost: 0 bytes in 0 blocks
==194894==    still reachable: 310,971 bytes in 3,042 blocks
==194894==         suppressed: 0 bytes in 0 blocks
==194894== 
==194894== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
